### DISCUSSION TITLE
Insufficient Data Usability with a single encoding

### DISCUSSION THREAD

**Discussion by MayteTDGeograma** [gopeg/issues-barriers#36] [General]
INSPIRE should be able to support other formats that are more suitable for working with large volumes of data, such as geopackage.

**Comment by sMorrone** [comment]
Dear @MayteToscanoDominguez 
this has indeed been a longstanding issue in the INSPIRE context and has led to the "Good Practice proposal for the GeoPackage encoding of INSPIRE datasets" (relevant specification can be found at https://github.com/INSPIRE-MIF/gp-geopackage-encodings/blob/main/spec/GeoPackage_Good_Practice_initiation_fiche.md).  With the "go-depth" use case we supported the proposal and contributed to the drafting of the related specifications, together with the EEA.
We addressed the Geology data theme. Our work has been successfully presented during the INSPIRE webinar on GeoPackage encoding of INSPIRE datasets (https://inspire.ec.europa.eu/events/inspire-good-practice-geopackage-and-implementation-practice-webinar). 
Good Practice is in its last mile, still pending only the final endorsement by the INSPIRE MIG that should come (hopefully) by the end of this year.

**Comment by sMorrone** [comment]
It is important to highlight that JRC will not provide the GeoPackage logical models and the related GeoPackage templates for all data themes. So, if a data provider /community is interested in delivering INSPIRE datasets in GeoPackage format for a specific data theme (e.g. BU) and related GeoPackage model and template do not exist, this data provider must create them and demonstrate they are conformant to the Good Practice specification.

**Comment by thorsten-reitz** [comment]
The Good Practice for GeoPackage has now been adopted.

**Comment by naomithiru** [comment]
This solution has been particularly useful for the Tracing use case due to the size reduction/faster rendering in desktop GIS and particularly for delivery of whole datasets which is optimal for network-type datasets.

**Comment by AndersOstmanNovogit** [comment]
This issue is very similar to issue #31. I have included your comments and solutions into section 4.2.2 of D4.3.

**Reply by AndersOstmanNovogit** [reply]
Now moved to D4.3 section 4.1.1
